tests/integration/bond_test.py

```diff
@@ -71,6 +73,7 @@ def test_remove_bond_with_minimum_desired_state():
     assert not state[INTERFACES]
 
 
+@pytest.mark.xfail
```

Comment:
You should move this to the previous commit.
Also, make sure you give the reason (like a link to the bug) and make it strict.